plugin-base/src/main/java/io/spine/tools/gradle/ProtocConfigurationPlugin.java

```diff
@@ -121,11 +122,11 @@ private void configurePlugins() {
         }
 
         private void configureProtocTasks(GenerateProtoTaskCollection tasks) {
-            Action<GenerateProtoTask> taskAction = task -> {
-                configureDescriptorSetGeneration(task);
-                plugin.customizeTask(task);
-            };
-            tasks.all().configureEach(taskAction);
+            var protocTasks = ImmutableList.copyOf(tasks.all());
```

Comment:
What was the nature of the issue this code fixes? Can we save this knowledge in a form of documentation of this method?

Reply:
The doc needs to answer at least to this question “Why is it correct doing the way now, rather than doing it the way we did before?”

I had a presumably similar issue from Kotlin when configuring an action for `GenerateProtoTasksCollection`.